postgres/tests/test_pg_integration.py

```diff
@@ -344,7 +344,7 @@ def test_statement_metrics_with_duplicates(aggregator, integration_check, pg_ins
     query_signature = 'a478c1e7aaac3ff2'
     normalized_query = 'select * from pg_stat_activity where application_name = ANY(array [ ? ])'
 
-    def obfuscate_sql(query):
+    def obfuscate_sql(query, options=None):
```

Comment:
Don't you think that it would be worth having a test with the new option set?

Reply:
Totally, however, in order to create a proper test for methods that interact with C bindings that communicate with the agent directly would require a hard dependency (rtloader) which would have some overhead to do because it would need to touch the build system in some way. Methods like this ([agent stub methods](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/stubs/datadog_agent.py#L7)) are difficult to test because of the disconnect, but I agree that it would be nice, it's just currently not worth it as we'd like to get this in before the 7.30 freeze.